### DISCUSSION TITLE
I need help please for right resize a image (4:3)

### DISCUSSION THREAD

**Discussion by Venus642** [ImageMagick/ImageMagick#4507] [Digital Image Processing] [answered]
Hello. (Problem picture resize. 4:3)
 
This is the original picture:
finish

New - But this is not right:
Code:
`magick 1_original.jpg -gravity center -extent "%[fx:w<h&&w/3>h/4?h/4*3:w]x%[h]" -extent "%[fx:w>h&&w/4>h/3?h/3*4:w]x%[h]" "2_new.jpg"`
finish

I'm look for this. Please. What is the right ImageMagick command?
4:3 ASPECT RATIO (70% of view-port of screens of any width)
finish

**Comment by GeeMack** [comment, marked as answer]
Using ImageMagick v7 cropping to a specific ratio can be done as simply as this...

    magick inimage.png -extent 4:3 outimage.png

The command you've shown should do what you want if you remove the `-gravity center` option. That command was written to crop a given aspect from the center of any input with unknown dimensions, and was probably originally written before ImageMagick had the aspect ratio option for "crop" and "extent".

**Comment by Venus642** [comment]
GeeMack! Big big thanks for your help.
That's what I've been looking for.
